cpp/dolfinx/fem/Form.h

```diff
@@ -277,6 +292,43 @@ class Form
       throw std::runtime_error("No mesh entities for requested domain index.");
   }
 
+  /// @brief Compute the list of entity indices for the ith integral (kernel)
+  /// of a given type. The indices are mapped from the integration domain mesh
```

Comment:
'of a given type' isn't clear. Does it mean same kernel, or same integration domain type?